doc/fping.pod

```diff
@@ -179,6 +179,10 @@ Calculate "outage time" based on the number of lost pings and the interval used
 Set the typ of service flag (TOS). I<N> can be either decimal or hexadecimal
 (0xh) format.
 
+=item B<--print-tos>
+
+Displays the TOS value in the output. However, the value is only displayed if a TOS value is present in the IP header.
```

Comment:
Any IPv4 packet always has a TOS byte value (or rather DSCP + ECN field values nowadays) in the IP header. But `fping` can only read when using raw sockets, and currently only for IPv4. I would suggest to write something like: "IPv4 only, requires root privileges or cap_net_raw."